interpreter/interpretable.go

```diff
@@ -114,6 +114,8 @@ type evalTestOnly struct {
 	attr  InterpretableAttribute
 	qual  Qualifier
 	field types.String
+	// hack attribute to preserve runtime cost behavior (see CostObserver for more)
+	protoHack bool
```

Comment:
Maybe don't call this out as a hack in the name of the field? Doing so does not aid the readability of the code, and unless we have some plan to eliminate this hack later, I'd prefer to make this edit as clean as possible. The purpose of this field is to identify that the `field` field is a protobuf field. Maybe something like `isProtoField` ?

Reply:
Makes sense - I've renamed it to `isProtoField`.